vowpalwabbit/search_dep_parser.cc

```diff
@@ -93,9 +93,21 @@ void initialize(Search::search &sch, size_t & /*num_actions*/, options_i &option
   all.pairs.swap(newpairs);
   all.triples.swap(newtriples);
 
+  std::vector<std::vector<uint8_t>> newpairs_(19);
```

Comment:
Can you please explain where these magic numbers come from?

Reply:
These are the lengths of the c-style arrays `pairs` and `triples` declared few lines above